### ISSUE TITLE
Export untranslated keys

### ISSUE THREAD

**Issue by Nonouf** [texterify/texterify#90]
It'd be great to be able to easily export untranslated keys and their respecting values, because unfortunately not every companies have people using the tools.

For example we use Texterify only within our development team to manage and synchronize our strings. When we need translations, we have to send a excel file to another team (yeah I know, very old school), who sends that back translated. We have a project in French and Dutch. We most of the time got our strings in French, which we need to send for translation in NL. Exporting untranslated keys in csv or json would be very helpful when we have a lot of strings to translate.

**Comment by chrztoph** [comment]
Hey,

I really like that idea. Maybe just make it a simple checkbox that says "Only export untranslated keys". What about context for the translators if they are not on Texterify. They should still get all keys for the default languages I guess? What do you think about that?

**Comment by Nonouf** [comment]
Yes, it could be a simple button or even an export config if you want to keep the same export system you already have. Just a guess, but using the export config would mean you could easily integrate it within the API without any extra work ;) 

Yes I think every existing data would be important. For example, for each key we should be able to get : 
* Key
* Description
* String in the default language

Not sure it'd be useful to return the other languages as well. Most of the time you have the string in the default language, then you ask for all the translations you need.

Finally, I wonder if it'd be useful to have overwrites as well in there. Not sure. I don't have a real-life use case.

What's you thoughts ?

**Comment by chrztoph** [comment]
Yeah sounds good. Added it to our roadmap. 🙂
